cli/utils.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import pathlib
+import click
+from libs import github_utils
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group("utils")
+def main():
+    pass
+
+
+@main.command()
+@click.argument("run-number", type=int, required=False)
+@click.option(
+    "--github-token",
```

Comment:
Thinking aloud... I'm not sure why, but my impression is values in the env are harder to steal than in args... though I guess if someone can read your command line history your machine is already owned and they can likely read the `export GITHUB_TOKEN=...`.